app/helpers.php

```diff
@@ -19,6 +19,18 @@ function api_version(): int
     return $version;
 }
 
+function array_filter_set(array|ArrayAccess $array): array
```

Comment:
should probably call this `array_filter_isset` or `array_reject_null` or something else; currently it sounds like this does something with Sets or is assigning something